girder/utility/plugin_utilities.py

```diff
@@ -35,11 +35,14 @@
 import sys
 import traceback
 import yaml
+import importlib
+
+from pkg_resources import iter_entry_points
 
 from girder.constants import PACKAGE_DIR, ROOT_DIR, ROOT_PLUGINS_PACKAGE, \
     TerminalColor
 from girder.models.model_base import ValidationException
-from girder.utility import config, mail_utils, mkdir
+from girder.utility import config as _config, mail_utils, mkdir
```

Comment:
My least favorite part of programming is naming things.  :/

My only thought was that many decorators are named so that they describe the function they are wrapping.  `@endpoint` means the the wrapped function is an endpoint.  I was thinking something like `@loader`.  I'm not sure that I agree that `config` -> `pluginConfig` is very beneficial.  The namespace should provide the specificity not the symbol name.  I.e. if I could totally rearrange everything, I might do something like this:
```python
from girder import plugin

@plugin.loader(config)
def loader(info):
    pass
```

That works equally well with `girder.plugin.config`,